core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -152,7 +152,9 @@
 
       it "generates the correct total amount" do
         order.cancellations.short_ship([inventory_unit_1])
+        inventory_unit_2.line_item.reload # UnitCancel#compute_amount needs updated amounts
         order.cancellations.short_ship([inventory_unit_2])
+        line_item.reload
```

Comment:
I really want to do this. I don't think we should concern ourselves with someone attaching adjustments to something other than LineItems and Shipments, since the rest of the methods in `OrderUpdater` won't take that into account.